assets/js/components/ImageRadio.js

```diff
@@ -80,7 +81,12 @@ export default function ImageRadio( props ) {
 					checked={ checked }
 					tabIndex={ tabIndex }
 				/>
-				<div className="mdc-image-radio__background">
+				<div
+					className={ classnames( 'mdc-image-radio__background', {
+						'mdc-image-radio__background--checked': checked,
+					} ) }
+					style={ { '--border-color': checkedBorderColor } }
```

Comment:
Not sure if an unprefixed variable name is best here, but given that it's applied as an inline style, it should be fine for now.